### PR TITLE
GoogleMaps: Channel parameter

### PR DESCRIPTION
**Pull request by dkemper**
You can add an optional parameter to the google maps requests to see the usage on a specific channel.

I added this parameter in this PR.
I also made some code style fixes. I hope its useful for you.

The description is here.
https://developers.google.com/maps/premium/overview?hl=en

### CODE REVIEW

File: src/Provider/GoogleMaps/Model/GoogleAddress.php [hunk 1]
```diff
@@ -483,15 +483,15 @@ public function getSubLocalityLevels()
...
+    public function withSubLocalityLevels(array $subLocalityLevel): GoogleAddress
     {
         $subLocalityLevels = [];
         foreach ($subLocalityLevel as $level) {
             if (empty($level['level'])) {
                 continue;
             }
 
-            $name = $level['name'] ?? $level['code'] ?? null;
+            $name = $level['name'] ?? $level['code'] ?? '';
```
Comment (Nyholm) [inline src/Provider/GoogleMaps/Model/GoogleAddress.php]: Why was this line changed?

File: src/Provider/GoogleMaps/Tests/GoogleMapsTest.php [hunk 2]
```diff
@@ -282,6 +284,33 @@ function (RequestInterface $request) use (&$uri) {
...
+            null,
+            null,
+            'bar'
+        );
+
+        try {
+            $provider->geocodeQuery(GeocodeQuery::create('blah'));
+        } catch (InvalidServerResponse $e) {
+        }
+        $this->assertEquals(
```
Comment (Nyholm) [inline src/Provider/GoogleMaps/Tests/GoogleMapsTest.php]: Maybe do a `asserRegex` instead. This test is likely to change when we add a new parameter.

File: src/Provider/GoogleMaps/Model/GoogleAddress.php (hunk 1 above)
Comment (dkemper) [reply src/Provider/GoogleMaps/Model/GoogleAddress.php]: new AdminLevel() $name parameter requires a string. Its never called because of the empty check before, but the given name parameter should be a string. 
It doesnt matter, but my editor shows it as wrong.

Comment (Nyholm) [reply src/Provider/GoogleMaps/Model/GoogleAddress.php]: Great. Thank you.

File: src/Provider/GoogleMaps/Tests/GoogleMapsTest.php (hunk 2 above)
Comment (dkemper) [reply src/Provider/GoogleMaps/Tests/GoogleMapsTest.php]: I just copied it from the other test. I can replace both if you want.
